envoy/stream_info/filter_state.h

```diff
@@ -16,6 +16,11 @@
 namespace Envoy {
 namespace StreamInfo {
 
+/*
+ * TODO (Jojy): Move all keys to a common file?
```

Comment:
Looks like in at least one other use case, the Filterstate::Object had 
 static const std::string& key();

which is nice because then only components who care about that object pull that key.  WDYT?